lib/pedicel/base.rb

```diff
@@ -171,18 +170,19 @@ def self.verify_x509_chain(root:, intermediate:, leaf:)
       raise SignatureError, 'invalid chain of trust' unless valid_chain
     end
 
-    def self.verify_signed_time(signature:, now:)
+    def self.verify_signed_time(signature:, now:, config: Pedicel.config)
```

Comment:
Maybe also `s/Pedicel.config/config/` here, like it has been changed in other places?

Reply:
The plan was to have `config: Pedicel.config` as default for functions that require it, which means functions use `config` to refer to the parameter, which by default is `Pedicel.config`.

Reply:
I would prefer letting it be `Pedicel.config`, simply because it's easier to see where it comes from. What do you think?